diff --git a/node_modules/@vaadin/router/dist/vaadin-router.js b/node_modules/@vaadin/router/dist/vaadin-router.js
index 70de26f..d2f72ab 100644
--- a/node_modules/@vaadin/router/dist/vaadin-router.js
+++ b/node_modules/@vaadin/router/dist/vaadin-router.js
@@ -239,6 +239,8 @@ function vaadinRouterGlobalClickHandler(event) {
     return;
   }
 
+  if (event && event.type === 'click') { window.scrollTo(0, 0); }
+
   // if none of the above, convert the click into a navigation event
   const {pathname, search, hash} = anchor;
   if (fireRouterEvent('go', {pathname, search, hash})) {
